config.yaml

```
@@ -0,0 +1,8 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  subject:
+    type: string
+    default: ubuntu.com
```

I don't think we want to use `ubuntu.com` here.
```suggestion
    default: tls-requirer-operator
```